Examples/AdvancedQueryOperations.md

```diff
@@ -7,6 +7,7 @@ Aggregation and other more complex RediSearch queries
     1.  [Data Load](#vss_dataload)
     2.  [Index Creation](#vss_index)
     3.  [Search](#vss_search)
+    4.  [Hybrid query Search](#vss_hybrid_query_search) 
```

Comment:
IMHO sort these within the topic, alphabetically for easier discovery. I like the table!